xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
If a correct, robust and well tested API changes, I agree 100%. If some flaws are obvious (and this one looks like some copy-and-paste error because "credits" isnt used as a key anywhere else) I tend to think differently. ;)